src/test/java/marquez/db/dao/TestOwnerDAO.java

```diff
@@ -0,0 +1,21 @@
+package marquez.db.dao;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import marquez.api.Owner;
+import marquez.db.dao.fixtures.DAOSetup;
+
+public class TestOwnerDAO {
+
+	@ClassRule
+	public static final DAOSetup daoSetup = new DAOSetup();
+
+	final OwnerDAO ownerDAO = daoSetup.onDemand(OwnerDAO.class);
+
+	@Test
+	public void testUpdateOwner() {
+		ownerDAO.insert(new Owner("Amaranta"));
```

Comment:
We should verify the result.

Reply:
agreed :) The DAO doesn't have a "get" yet so I stopped there. It doesn't fail which mean that the db is setup properly. I wanted to have the DB setup to bootstrap DAO testing.